Formula/nginx.rb

```diff
@@ -40,17 +50,16 @@ def install
     end
 
     pcre = Formula["pcre"]
-    openssl = Formula["openssl"]
-    libressl = Formula["libressl"]
 
-    if build.with? "libressl"
-      cc_opt = "-I#{pcre.include} -I#{libressl.include}"
-      ld_opt = "-L#{pcre.lib} -L#{libressl.lib}"
+    if build.head?
+      openssl = Formula["openssl@1.1"]
```

Comment:
@MikeMcQuaid I thought you hated ternary operators so I didn't suggest it 😄, but if you're happy I'm happy for OP to use that.